andela/bp-esa-frontend
# Feature #160437219 Feedback on the search box

**What does this PR do?**
Implements the feedback given on the search box.

**Description of Task to be completed?**
Adjusting the size of the search box and the margin alignment.

**How should this be manually tested?**
Clone and set up the project using the instruction in the README.
Use `$ git checkout feedback-on-search-box `to check out the branch containing the changes of this pull request
Use `$ npm start` to start the application
Visit http://localhost:3000/ to see the report page.
When the page is loaded the search box is aligned with the automation status drop-down and same size as the search with drop-down.
What are the relevant pivotal tracker stories?
[#160437219](https://www.pivotaltracker.com/story/show/160437219)

**Screenshots (if appropriate)**
<img width="1440" alt="screen shot 2018-11-27 at 11 20 56" src="https://user-images.githubusercontent.com/6957960/49068629-3f5dc380-f238-11e8-92c5-93756309b03b.png">
